spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/RemoteTokenServices.java

```diff
@@ -109,6 +109,13 @@ public OAuth2Authentication loadAuthentication(String accessToken) throws Authen
 		headers.set("Authorization", getAuthorizationHeader(clientId, clientSecret));
 		Map<String, Object> map = postForMap(checkTokenEndpointUrl, formData, headers);
 
+		if (map == null) {
```

Comment:
Can you change this to `org.springframework.util.CollectionUtils.isEmpty()`

Reply:
> 你能改变成 `org.springframework.util.CollectionUtils.isEmpty()`

Ok, it has been modified and added test